### ISSUE TITLE
WTabSet keyboard interaction inaccurate

### ISSUE THREAD

**Issue by marksreeves** [BorderTech/wcomponents#692]
Tabs should activate on keyboard navigation according to the [WAI-ARIA Authoring Practices](http://www.w3.org/TR/wai-aria-practices/#tabpanel):

> Left Arrow - with focus on a tab, pressing the left arrow will move focus to the previous tab in the tab list and activate that tab. Pressing the left arrow when the focus is on the first tab in the tab list will move focus and activate the last tab in the list.
Right Arrow - with focus on a tab, pressing the right arrow will move focus to the next tab in the tab list and activate that tab. Pressing the right arrow when the focus is on the last tab in the tab list will move focus to and activate the first tab in the list.

**Comment by marksreeves** [comment]
In addition TabMode.SERVER is now unusable and fails a11y by causing an unexpected change of context. To be able to support rich content in tabs we must remove TabMode.SERVER. Doing this will also improve the UX for these tabs. In this release TabMode.Server will be silently mapped to TabMode.DYNAMIC.
